package.json

```diff
@@ -10,7 +10,7 @@
   ],
   "scripts": {
     "build": "run-p --print-label build:lyra-api build:docs build:extension",
-    "build:netlify": "run-s build:lyra-api build:docs",
+    "build:netlify": "cd docs && npm i && npm run build",
```

Comment:
using npm because using yarn defaults to using workspaces, which we don't want to use for docs. just install everything from the registry for docs